internal/mode/static/handler.go

```diff
@@ -111,16 +113,20 @@ func (h *eventHandlerImpl) HandleEventBatch(ctx context.Context, logger logr.Log
 		return
 	case state.EndpointsOnlyChange:
 		h.cfg.version++
+		cfg := dataplane.BuildConfiguration(ctx, graph, h.cfg.serviceResolver, h.cfg.version)
+		h.latestConfiguration = &cfg
```

Comment:
Can this ever set latestConfiguration to an errored/invalid configuration, thus messing up telemetry?